### PR TITLE
splashtop-business-access: Add version 3.3.8.1

### PR DESCRIPTION
**Pull request by mastqe**

### CODE REVIEW

File: bucket/splashtop-business-access [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    ],
+    "checkver": {
+        "url": "https://www.splashtop.com/wp-content/themes/responsive/downloadx.php?product=stb&platform=windows-client-portable",
+        "jsonpath": "$.url",
+        "regex": "(?<URL>https:.*_v(?<version>\\d+\\.\\d+\\.\\d+\\.\\d+)\\.exe)"
+    },
+    "autoupdate": {
+        "url": "$matchURL"
+    }
+}
```
Comment (Ash258) [inline bucket/splashtop-business-access:27]:
```suggestion
{
    "version": "3.3.8.1",
    "description": "Fast, simple, secure remote computer access for individuals and teams",
    "homepage": "https://www.splashtop.com/business",
    "license": {
        "identifier": "Proprietary",
        "url": "https://www.splashtop.com/terms/splashtop"
    },
    "url": "https://d17kmd0va0f0mp.cloudfront.net/winclient/STB/Splashtop_Business_Win_PORTABLE_v3.3.8.1.exe#/splashtop.exe",
    "hash": "00d7e7da70744d57301e6fd855c048e8ec1a36df3efd76e7f0500929cf2aef23",
    "shortcuts": [
        [
            "splashtop.exe",
            "Splashtop Business Access"
        ]
    ],
    "checkver": {
        "url": "https://www.splashtop.com/wp-content/themes/responsive/downloadx.php?product=stb&platform=windows-client-portable",
        "jsonpath": "$.url",
        "regex": "([\\d.]+)\\.exe"
    },
    "autoupdate": {
        "url": "https://d17kmd0va0f0mp.cloudfront.net/winclient/STB/Splashtop_Business_Win_PORTABLE_v$version.exe#/splashtop.exe"
    }
}

```

Comment (Ash258) [reply bucket/splashtop-business-access]: ...

Comment (mastqe) [reply bucket/splashtop-business-access]: oops. Totally missed those changes, just thought the whole file changed from the mv... Should be fixed now.

**Pull request merged**